<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>groupFile</class>
 <widget class="QGroupBox" name="groupFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>135</height>
   </rect>
  </property>
  <property name="title">
   <string>File</string>
  </property>
  <layout class="QGridLayout" name="gridFile">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formExp">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridExp">
       <item row="0" column="0">
        <widget class="QLineEdit" name="lineExp">
         <property name="toolTip">
          <string>The directory where the recordings will be saved</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="buttonBrowse">
         <property name="toolTip">
          <string>The directory where the recordings will be saved</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelExp">
       <property name="toolTip">
        <string>The directory where the recordings will be saved</string>
       </property>
       <property name="text">
        <string>Experiment directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineFileName">
       <property name="toolTip">
        <string>The name of this particular recording within the experiment</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelFileName">
       <property name="toolTip">
        <string>The name of this particular recording within the experiment</string>
       </property>
       <property name="text">
        <string>Trial name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboFileType">
       <property name="toolTip">
        <string>The file type of saved recordings</string>
       </property>
       <item>
        <property name="text">
         <string>.npy</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.mat</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelFileType">
       <property name="toolTip">
        <string>The file type of saved recordings</string>
       </property>
       <property name="text">
        <string>File type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="labelMaxFileSize">
       <property name="toolTip">
        <string>The size of the files that recordings will be broken into</string>
       </property>
       <property name="text">
        <string>Maximum file size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="comboFileSize">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>250 MB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500 MB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 GB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 GB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 GB</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
